Lib/test/test_tkinter/widget_tests.py

```diff
@@ -312,8 +356,11 @@ def test_configure_font(self):
                         '-Adobe-Helvetica-Medium-R-Normal--*-120-*-*-*-*-*-*')
         is_ttk = widget.__class__.__module__ == 'tkinter.ttk'
         if not is_ttk:
-            self.checkInvalidParam(widget, 'font', '',
-                                   errmsg='font "" doesn\'t exist')
+            if tk_version < (9,0):
+                errmsg = 'font "" doesn\'t exist'
+            else:
+                errmsg = 'font "" does not exist'
```

Comment:
Nit: `checkInvalidParam` uses `assertRaisesRegex`, so we should be able to simplify:
```suggestion
            errmsg = 'font "" does ?n[o\']t exist'
```

Reply:
OK